nmdc_server/config.py

```diff
@@ -15,6 +15,10 @@ class Settings(BaseSettings):
     ingest_database_uri: str = "postgresql:///nmdc_testing"
     testing_database_uri: str = "postgresql:///nmdc_testing"
 
+    # database tuning knobs
+    db_pool_size: int = 20
```

Comment:
These numbers are still pretty conservative for the hardware we're running on. Once this is deployed I'll mark the sentry errors as resolved so we'll be notified if they crop up again.